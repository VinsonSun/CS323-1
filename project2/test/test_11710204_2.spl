int test_11710204_r2()
{
  c = '1';
}
