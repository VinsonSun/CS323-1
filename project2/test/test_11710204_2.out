Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: test_11710204_r2
        LP
        RP
      CompSt (2)
        LC
        StmtList (3)
          Stmt (3)
            Exp (3)
              Exp (3)
                ID: c
              ASSIGN
              Exp (3)
                CHAR: '1'
            SEMI
        RC
